src/main/java/com/google/impactdashboard/database_manager/FakeDatabase.java

```diff
@@ -0,0 +1,216 @@
+package com.google.impactdashboard.database_manager;
+
+import java.lang.Math;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import com.google.impactdashboard.data.recommendation.*;
+import com.google.impactdashboard.data.project.ProjectIdentification;
+import com.google.impactdashboard.data.IAMBindingDatabaseEntry;
+import java.util.concurrent.atomic.AtomicReference;
+
+/** A class that maintains a fake database for testing purposes. */
+public class FakeDatabase {
+  private static final long MILLISECONDS_365_DAYS = Long.parseLong("31536000000");
+  private static final long MILLISECONDS_ONE_DAY = 86400000;
+
+  /** 
+   * Represents a database table that holds information about accepted 
+   * recommendations. 
+   */
+  private static Map<Long, Recommendation> recommendations = 
+    new HashMap<Long, Recommendation>() {
+        private static final long serialVersionUID = 1L;
+
+        {
+        put(Long.parseLong("1592486705000"), 
```

Comment:
Here and throughout, you should be able to do 1592486705000L, if I'm not mistaken.